src/app/workspaces/workspace-details/workspace-details.controller.js

```diff
@@ -55,9 +55,10 @@ class WorkspaceDetailsCtrl {
       this.updateWorkspaceData();
     }
 
-
+    this.startUpdateWorkspaceStatus();
```

Comment:
I would link promise of the status to the loading progress

Else we may have loading = false while status is not yet there

so loading = false should occur when we get both details and status